crates/turborepo-ui/src/tui/mod.rs

```diff
@@ -36,4 +37,6 @@ pub enum Error {
     Stdin { name: String, e: std::io::Error },
     #[error(transparent)]
     Io(#[from] std::io::Error),
+    #[error("Unable to persist preferences")]
```

Comment:
Would it make sense to do:

```suggestion
    #[error("Unable to persist preferences. Please file a bug report.")]
```